resiliont/.htaccess

```diff
@@ -2,4 +2,4 @@ Options -MultiViews
 RewriteEngine on
 
 RedirectMatch 302 ^/resiliont/(home|git|repo)/?$  https://github.com/pedropaulofb/resiliont/
-RewriteRule ^resiliont/er2024paper/?$ https://raw.githubusercontent.com/pedropaulofb/resiliont/main/resources/Ontological%20Foundations%20of%20Resilience.pdf [R=302,NE,L]
+RewriteRule ^/resiliont/er2024paper/?$ https://raw.githubusercontent.com/pedropaulofb/resiliont/main/resources/Ontological%20Foundations%20of%20Resilience.pdf [R=302,NE,L]
```

Comment:
I think the fix you are looking for is:
`RewriteRule ^er2024paper/?$ https://raw.githubusercontent.com/pedropaulofb/resiliont/main/resources/Ontological%20Foundations%20of%20Resilience.pdf [R=302,NE,L]` Right now you are minting `w3id.org/resiliont/resiliont/er2024paper`